debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/ApicurioRegistryContainer.java

```diff
@@ -5,20 +5,69 @@
  */
 package io.debezium.testing.testcontainers;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.net.ssl.HttpsURLConnection;
+
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
 
 public class ApicurioRegistryContainer extends GenericContainer<ApicurioRegistryContainer> {
 
-    private static final String DEBEZIUM_VERSION = DebeziumContainer.getDebeziumStableVersion();
+    private static final String APICURIO_VERSION = getApicurioStableVersion();
     private static final Integer APICURIO_PORT = 8080;
 
     public ApicurioRegistryContainer() {
-        super("apicurio/apicurio-registry-mem:" + DEBEZIUM_VERSION);
+        super("apicurio/apicurio-registry-mem:" + APICURIO_VERSION);
 
         this.waitStrategy = new LogMessageWaitStrategy()
                 .withRegEx(".*apicurio-registry-app.*started in.*");
 
         addExposedPort(APICURIO_PORT);
     }
+
+    public static String getApicurioStableVersion() {
+        String APICURIO_VERSION = "";
```

Comment:
This is excellent! But as it already happened in the past it is possible that Apicurio might be broken. Could you please add a support for a test system property (prefixed with `test.`)? When set the version passed via sysproterty will be used, otherwise this code will be executed.